lib/bots/versionbot.ts

```diff
@@ -769,9 +769,9 @@ export class VersionBot extends ProcBot {
         };
 
         return Promise.mapSeries([
-            `git clone https://${versionData.authToken}:${versionData.authToken}@github.com/` +
-                `${versionData.repoFullName} ${versionData.fullPath}`,
-            `git checkout ${versionData.branchName}`
+            `git clone "https://${versionData.authToken}:${versionData.authToken}@github.com/` +
+                `${versionData.repoFullName}" ${versionData.fullPath}`,
+            `git checkout "${versionData.branchName}"`
```

Comment:
@jviotti Would switching from `exec` to `spawn` also fix some of the quoting issues we had on Etcher? Or do our problems run much deeper than that? ;-)